src/qililab/calibration/calibration_node.py

```diff
@@ -360,7 +360,7 @@ def run_node(self) -> float:
 
         # When notebook execution fails, generate error folder and move there the notebook:
         except Exception as exc:  # pylint: disable = broad-exception-caught
-            if output_path in [os.scandir(os.getcwd())]:
+            if output_path in [entry.path for entry in os.scandir(os.getcwd())]:
```

Comment:
## Update after testing:
- The errors are being created on the error folder
- And they are being uploaded to AWS
<img width="541" alt="Screenshot 2024-08-13 at 18 10 19" src="https://github.com/user-attachments/assets/4c061622-4ac7-4390-b6de-49ba21871354">
<img width="1051" alt="Screenshot 2024-08-13 at 18 10 25" src="https://github.com/user-attachments/assets/95bb1fd3-1a21-470a-bfd9-b03d430721bb">

Reply:
@IsaacLA12 I just removed the error_execution folder creation, since I think its less intuitive, and practical, and also we don't need to worry about this anymore 👍